src/org/authInfo.ts

```diff
@@ -373,40 +385,47 @@ export class AuthInfo extends AsyncOptionalCreatable<AuthInfo.Options> {
   /**
    * Get a list of all authorizations based on auth files stored in the global directory.
    *
-   * @returns {Promise<SfOrg[]>}
+   * @returns {Promise<OrgAuthorization[]>}
    */
-  public static async listAllAuthorizations(): Promise<SfOrg[]> {
+  public static async listAllAuthorizations(): Promise<OrgAuthorization[]> {
     const globalInfo = await GlobalInfo.getInstance();
+    const config = (await ConfigAggregator.create()).getConfigInfo();
     const auths = Object.values(globalInfo.orgs.getAll());
-    const final: SfOrg[] = [];
+    const final: OrgAuthorization[] = [];
     for (const auth of auths) {
       const username = ensureString(auth.username);
-      const alias = globalInfo.aliases.get(username) ?? undefined;
+      const aliases = globalInfo.aliases.getAll(username) ?? undefined;
+      const configs = config
+        .filter((c) => aliases.includes(c.value as string) || c.value === username)
+        .map((c) => c.key);
```

Comment:
Can we put a comment here about what exactly this is supposed to be doing please?